### PR TITLE
Adding tabbed navigation to the project page

### LINKED ISSUE #1100: Add tabbed navigation to the project page
Figma link: https://www.figma.com/design/rgzEYTpnGl6l68moRSVwYg/Phase-1-UI-Details?node-id=1325-22724&node-type=frame&t=etwn2DWmZNdNuUnP-0

Referenced in https://github.com/pulibrary/tigerdata-app/issues/1081

Add tabbed navigation to the project show page to navigate between sections as shown here:

388791655-6e319aaa-7c62-4c33-aad1-050be3b5d280 (1)

The tab below follows the same navigation format as the dashboard, referencing issue #7 , #15 

Screenshot 2024-11-21 at 11 03 00 PM

Mockup Figma Link: https://www.figma.com/design/rgzEYTpnGl6l68moRSVwYg/Phase-1-UI-Details?node-id=1325-22724&t=MJxVuyRzH5BKs6pB-4

switch to dev mode for detailed spacing info and CSS code

## Acceptance criteria

- [ ] Remove all buttons except the "Approve" button from the bottom of the show page
- [x] There is tabbed navigation for "Content Preview" and "Details" only between sections of a project's show page
- [ ] There is a test for this

### PR DESCRIPTION
**Pull request by JaymeeH**
closes #1100

### CODE REVIEW

**Comment by JaymeeH** [comment]
Screenshot 2024-12-11 at 4 39 57 PM
Screenshot 2024-12-11 at 4 40 01 PM

File: app/views/projects/contents.html.erb [hunk 1]
```diff
@@ -51,5 +51,7 @@
 </div>
 <script type="module">
     // Setup the DataTables
```
Comment (hectorcorrea) [inline app/views/projects/contents.html.erb:53]: We should remove this comment, it's misleading (and probably a copy and paste thing)

File: app/javascript/entrypoints/projectTabs.js [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+  const project = document.getElementById('show') || document.getElementById('content');
+
+  // Check the session to see which tab should start as active be active
+  if (project) {
+    switch (session) {
+      case 'content':
+        content.style.borderBottom = 'solid';
+        content.style.borderColor = '#E77500';
+        content.classList.add('active');
+        break;
```
Comment (hectorcorrea) [inline app/javascript/entrypoints/projectTabs.js:16]: These case statements are repeated, is there a way to remove the duplication?

**Review by hectorcorrea — APPROVED** [review]
Looks great @JaymeeH - I left a couple of comments with suggestions but it's up to you.

**Pull request merged**